src/components/ListItem.js

```diff
@@ -58,17 +58,15 @@ const ListItem = () => {
   const handleOnChange = (itemName) => {
     for (let item of items) {
       if (item.itemName === itemName) {
-        item.lastPurchase = Date.now();
-        const daysSinceLastTransaction = item.lastPurchasedDate
-          ? Math.round(
-              (Date.now() - item.lastPurchasedDate) / 1000 / 60 / 60 / 24,
-            )
+        const daysSinceLastTransaction = item.lastPurchase
+          ? Math.round((Date.now() - item.lastPurchase) / 1000 / 60 / 60 / 24)
```

Comment:
Do you know about the concept of **magic numbers** in programming? this **1000 / 60 / 60 / 24** for example can be a magic number, by avoiding them we can make our code more readable and maintainable, you have some info here https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
This is interesting, @llunaplanet. Thank you for the share!
I never heard this term before. I read the link that you gave, but still don't understand.
Would love to hear about it more tonight if it's possible! 😀